app/src/main/java/com/bucketlist/ui/main/MainFragment.kt

```diff
@@ -10,25 +10,25 @@ import com.bucketlist.R
 
 class MainFragment : Fragment() {
 
-    companion object {
-        fun newInstance() = MainFragment()
-    }
-
     private lateinit var viewModel: MainViewModel
 
     override fun onCreateView(
-        inflater: LayoutInflater, container: ViewGroup?,
-        savedInstanceState: Bundle?
-    ): View {
-        return inflater.inflate(R.layout.main_fragment, container, false)
-    }
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,) =
+            inflater.inflate(R.layout.main_fragment, container, false,)
+
 
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
         // TODO: Use the ViewModel
     }
 
+    companion object {
```

Comment:
Moving companion object to bottom of the class as recommended here: https://kotlinlang.org/docs/coding-conventions.html#class-layout